data/korath/korath missions.txt

```diff
@@ -257,3 +257,28 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "Korath Boarding Response"
+	invisible
+	boarding
+	repeat
+	minor
+	deadline 1
+	to offer
+		random < 5
+		or
+			"reputation: Korath" < -5000
+			"reputation: Korath (Civilian)" < -30
+		not "Korath Boarding Response: active"
+	source
+		government Korath "Korath (Civilian)"
+		system Alcyone Almach "Alpha Hydri" Atik Durax Misam Polaris Sheratan
+	on offer
+		conversation
+			"An angry hiss comes over your radio, and you see a strange pulse along the hull of the Korath vessel. Your ship detects a powerful hyperwave emission followed by several capital ships jumping in. You had best finish your business with this ship before yours is reduced to a ball of super-heated plasma."
```

Comment:
```suggestion
			"An angry hiss comes over your radio, and you see a strange pulse along the hull of the Korath vessel. Your ship detects a powerful hyperwave emission followed by several capital ships jumping in. You had best finish your business with this ship before yours is reduced to a ball of superheated plasma."
```

Reply:
I see both "super-heated" and "superheated" in articles on plasma. I'm indifferent, so I'll use your suggestion, but I'll leave this conversation unresolved in case someone has a more authoritative answer.